llvm/lib/Target/Xtensa/XtensaISelLowering.cpp

```diff
@@ -1104,10 +1104,24 @@ XtensaTargetLowering::emitSelectCC(MachineInstr &MI,
 MachineBasicBlock *XtensaTargetLowering::EmitInstrWithCustomInserter(
     MachineInstr &MI, MachineBasicBlock *MBB) const {
   DebugLoc DL = MI.getDebugLoc();
+  const TargetInstrInfo &TII = *Subtarget.getInstrInfo();
```

Comment:
Use XTensaInstrInfo. Also this might be a member already?

Reply:
I added using XtensaInstrInfo. It is not a member yet, I'm not sure, whether we need to add it?